src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift

```diff
@@ -66,6 +74,35 @@ extension Name {
 	}
 }
 
+fileprivate extension String {
+	
+	func groupingComponents() -> [String] {
+		// the string shall be trimmed for leading and training whitespace
+		let components: [String] =
+		self.trimmingCharacters(in: .whitespaces)
+		// the string shall be trimmed for leading and trailing `<`
+			.trimmingCharacters(in: CharacterSet(charactersIn: "<"))
+		// any whitespace in the string shall be replaced by `<`
+			.replacingOccurrences(of: "\\s+", with: "<", options: .regularExpression)
+		// any occurence of more than one `<` shall be replaced by a single `<`
+			.replacingOccurrences(of: "<+", with: "<", options: .regularExpression)
+			.replacingOccurrences(of: "-", with: "<")
+			.replacingOccurrences(of: ".", with: "<")
```

Comment:
Would it make sense to do these two first, before replacing multiple `<` with a single `<`? And maybe mention these steps in a comment as well

Reply:
done